packages/server/bootstrap.js

```diff
@@ -14,7 +14,14 @@ const appRoot = require('app-root-path')
 // If running in test env, load .env.test first
 // (appRoot necessary, cause env files aren't loaded through require() calls)
 if (isTestEnv()) {
-  dotenv.config({ path: `${appRoot}/.env.test` })
+  const { error } = dotenv.config({ path: `${appRoot}/.env.test` })
+  if (error) {
+    const e = new Error(
+      'Attempting to run tests without an .env.test file properly set up! Check readme!'
+    )
+    console.error(e)
+    process.exit(1)
```

Comment:
Exiting directly not throwing, so that the error doesn't get caught by yargs and reported as something else (happened to me with an old node_modules)